node/whitelist.go

```diff
@@ -0,0 +1,90 @@
+package node
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pokt-network/pocket-core/config"
+	"io/ioutil"
+	"sync"
+)
+
+type Whitelist struct{
+	list map[string]struct{}
+	mux sync.Mutex
+}
+
+var(
```

Comment:
Both whitelists are initialized exactly once, exactly on startup. Maybe we want to have a singly `sync.Once` object and initialize them both?